src/PowerShellEditorServices/Services/PowerShellContext/PowerShellContextService.cs

```diff
@@ -1143,12 +1155,12 @@ public async Task LoadHostProfilesAsync()
                 {
                     PSCommand command = new PSCommand();
                     command.AddCommand(profilePath, false);
-                    await this.ExecuteCommandAsync<object>(command, true, true);
+                    await this.ExecuteCommandAsync<object>(command, sendOutputToHost: true, sendErrorToHost: true).ConfigureAwait(false);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this default value assigned to parameter 'sendErrorToHost'.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=4812399)